package.json

```diff
@@ -64,14 +64,14 @@
     "redux-mock-store": "^1.2.2",
     "sass-loader": "~6.0.6",
     "style-loader": "^0.13.1",
-    "stylelint": "^9.3.0",
+    "stylelint": "^15.2.0",
     "stylelint-config-standard": "^18.0.0",
-    "uglifyjs-webpack-plugin": "^1.2.2",
+    "tabbable": "~5.2.0",
+    "uglifyjs-webpack-plugin": "^2.2.0",
     "url-loader": "^1.0.1",
-    "webpack": "^3.4.1",
+    "webpack": "^5.75.0",
```

Comment:
This is a separate change. Did you intend to include this?

Reply:
No, I have encountered a problem and while trying to solve it I tried updating some versions of some dependencies.
I just wanted to push these changes so @MariaAga could have a look at my local code as it is rn
I won't include the second commit in the final PR (including these changes)

(Converted to draft)

Reply:
Removed any package updates 👍🏼